Recap/UseCases/Summary/SummaryView.swift

```diff
@@ -5,6 +5,7 @@ struct SummaryView: View {
     let onClose: () -> Void
     @ObservedObject var viewModel: SummaryViewModel
     let recordingID: String?
+    @State var showingTranscript: Bool = false
```

Comment:
- Should be part of the view model 
- `showingTranscript` should be `showMeetingTranscript` 
- Scope should be private